src/Controller/QuestionController.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\QuestionManager;
+
+class QuestionController extends AbstractController
+{
+
+    /**
+     * @return string
+     * Displays a random question and its possible choices.
```

Comment:
It's probably more common to have your comments before the PHPDoc codes. Because you need to know what the method is for before how to use it